.github/workflows/benchmarks.yml

```
@@ -0,0 +1,108 @@
+name: Benchmarks
+
+# On PRs: compile benches (no-run) to catch regressions without DB.
+# On main: full bench run with a containerized MySQL service so
+#   rows_processing and value_conversion benches can execute. Results are
+#   uploaded as artifacts for trend inspection.
+
+on:
```

YAMLlint (1.38.0)

[warning] truthy value should be one of [false, true]

(truthy)